homeassistant/components/speedtestdotnet/sensor.py

```diff
@@ -54,10 +53,14 @@ def __init__(
         """Initialize the sensor."""
         super().__init__(coordinator)
         self.entity_description = description
-
         self._attr_name = f"{DEFAULT_NAME} {description.name}"
         self._attr_unique_id = description.key
         self._attrs = {ATTR_ATTRIBUTION: ATTRIBUTION}
+        self._attr_device_info = {
```

Comment:
As above.

Reply:
I added the device info option to show the integration as a service.

Reply:
Please add it in a separate PR.

Reply:
done